content/en/examples/application/mysql/mysql-configmap.yaml

```diff
@@ -10,8 +10,9 @@ data:
     # Apply this config only on the primary.
     [mysqld]
     log-bin
+    # datadir=/var/lib/mysql/mysql
   replica.cnf: |
     # Apply this config only on replicas.
     [mysqld]
     super-read-only
-
+    # datadir=/var/lib/mysql/mysql
```

Comment:
I'm a bit confused about how this resolves the issue? aren't theses just comments? can you provide that running the steps with updated configmap modified resolved the original issue?

Reply:
Sure i will paste the steps here.

Reply:
@Kartik494 , stilling working on this PR?
Will you paste steps here like mentioned [previously](https://github.com/kubernetes/website/pull/34694#discussion_r943086138)

Reply:
@reylejano i have been busy in doing other assignments, sure i will paste this soon.